changelog/13202.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
```

Comment:
To use the feature tag, this should be formatted for a new feature (see the wiki page). I _think_ this is more of an improvement though.

Reply:
I'm also not sure what the subsystem should be, exactly, but this isn't like what most subsystems look like.

Reply:
Realized in a later PR that we do not need a changelog request per PR. 
https://hashicorp.atlassian.net/wiki/spaces/VAULT/pages/1311244491/Changelog+Process#New-and-Major-Features.
Will be removing this file in next submission. Thanks.